# CellInfoList Message

std_msgs/Header header

openpilot/CellInfo[] cellInfo
